cmd/zt_block_size_to_bytes_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2017 Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	chk "gopkg.in/check.v1"
+)
+
+type blockSizeFilterSuite struct{}
+
+var _ = chk.Suite(&blockSizeFilterSuite{})
+
+func (s *genericFilterSuite) TestConversions(c *chk.C) {
+
+	testData := []struct {
```

Comment:
Minor nitpick: Inline struct definition.

Doesn't super duper matter since it's never used again but I'm also pretty sure this isn't idiomatic

Reply:
I copied it out of Dave Cheney's blog post on table driven tests in Go, so I guess that qualifies it as idiomatic ;-)   Will leave it here, but I don't think I use it outside of tests tho.